.code-samples.meilisearch.yaml

```diff
@@ -424,3 +431,61 @@ phrase_search_1: |-
 authentication_header_1: |-
   $ curl -X GET 'http://127.0.0.1:7700/keys' \
   --header "X-Meili-API-Key: mast
+sorting_guide_update_sortable_attributes_1: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/settings/sortable-attributes' \
+  --data '[
+    "author",
+    "price"
+  ]'
+sorting_guide_update_ranking_rules_1: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/settings/ranking-rules' \
+  --data '[
+    "words",
+    "sort",
+    "typo",
+    "proximity",
+    "attribute",
+    "exactness"
+  ]'
+sorting_guide_sort_parameter_1: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/search' \
+  --data '{
+    "q": "science fiction",
+    "sort": [
+      "price:asc"
+    ]
+  }'
+sorting_guide_sort_parameter_2: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/search' \
+  --data '{
+    "q": "butler",
+    "sort": [
+      "author:desc"
+    ]
+  }'
+get_sortable_attributes_1: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/books/settings/sortable-attributes'
+update_sortable_attributes_1: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/books/settings/sortable-attributes' \
+    --data '[
+        "price",
+        "author"
+    ]'
+reset_sortable_attributes_1: |-
+  curl \
+    -X DELETE 'http://localhost:7700/indexes/books/settings/sortable-attributes'
+search_parameter_guide_sort_1: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/books/search' \
+    --data '{
+      "q": "science fiction",
+      "sort": [
+        "price:asc"
+      ]
+    }'
```

Comment:
You should not put a new line at the end of the file. It does not impact any usage, but it's a style rule to avoid unrelated changes between two contributors in the future PRs

To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...) thanks to the .editorconfig file in this repo.
See more about editorconfig: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/documentation/blob/master/.editorconfig

Reply:
ah, sorry! I got carried away in the rush of things because we have a markdown lint rule that enforces new lines at the end of every .md (which is something I really want to change but haven't had the time so far).

Anyway, thanks, I'll take care of it =)